### PR TITLE
Streamlined setup process

### PR DESCRIPTION
**Pull request by cimermanGregor**
- Moved static files into project direcotry
- Moved AMQP broker to localhost, suggested usage of docker
- Updated toolson mail tests

### CODE REVIEW

**Comment by blazdivjak** [comment]
I guess we could create a makefile instead of a script to automate deployment.

**Comment by blazdivjak** [comment]
Would it be usefull to create a docker container for development instead of contaminating the base system?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+RUN pip install -r requirements.txt
+
+# prepare environment
+RUN cd ${automator_dir}
+RUN python manage.py reset_db --noinput --settings $settings
+RUN python manage.py migrate --settings $settings
+RUN python manage.py createcachetable --settings $settings
+RUN python manage.py test --settings $settings
+
+CMD cd lib/openconfig/ && ./openconfig.sh & python manage.py runserver 0.0.0.0:8000
```
Comment (blazdivjak) [inline Dockerfile]: Replace CMD with ENTRYPOINT?

File: scripts/bootstrap_project.sh [hunk 2]
```diff
@@ -72,5 +74,7 @@ python manage.py migrate --settings $SETTINGS
 #python manage.py collectstatic --noinput --settings $SETTINGS
 echo "DATABASE: Creating cache table"
 python manage.py createcachetable --settings $SETTINGS
-echo "DATABASE: Loading Initial data"
-echo "SET SAIL FOR ADVENTURE: Use username: admin, password: tMRAGkMDfMk3UJ2zJKgb to login in http://0.0.0.0:8000"
\ No newline at end of file
+# echo "DATABASE: Loading Initial data"
+# echo "SET SAIL FOR ADVENTURE: Use username: admin, password: tMRAGkMDfMk3UJ2zJKgb to login in http://0.0.0.0:8000"
+echo "Now create superuser: \"python manage.py createsuperuser --settings $SETTINGS\""
+echo "Then start the server: \"python manage.py runserver 0.0.0.0:8000 --settings $SETTINGS\""
```
Comment (blazdivjak) [inline scripts/bootstrap_project.sh:82]: This file looks redundent if we use docker containers.

**Comment by cimermanGregor** [comment]
Thanks! I will sort it out.

File: README.md [hunk 3]
```diff
@@ -140,6 +140,24 @@ _Additionaly based on serializer class defined for a specific view, different at
...
+docker run -itd -p 8000:8000 -v $(pwd):/opt/automator --name automator automator
+docker run -itd -v $(pwd):/opt/automator --name celery automator celery -A automator worker -l info
+docker run -itd -p 5555:5555 -v $(pwd):/opt/automator --name flower automator celery -A automator flower --port=5555
+```
+ 
+SQLite is used for database by default and it is mounted to development directory.
+
+Create superuser
+```
+docker exec -it automator python manage.py createsuperuser
```
Comment (blazdivjak) [inline README.md]: Should this be executed when you create a container?

File: Dockerfile (hunk 1 above)
Comment (cimermanGregor) [reply Dockerfile]: In this case I used CMD for when user mounts local folder to the container code folder. Openconfig script must be ran and the first part of the command solves this. I will setup ENTRYPOINT for working directory.

File: README.md (hunk 3 above)
Comment (cimermanGregor) [reply README.md]: Maybe. This command is interactive and user may specify it's own superuser profile so I think it is better if this is left to the developer to set it after the initialisation.

**Review by blazdivjak — APPROVED** [review]
LGTM. Merge!

**Pull request merged**